model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -990,6 +1035,12 @@ async def create_tensorrt_llm_bundle(
         ).model_bundle_id
 
 
+async def get_chat_template(
```

Comment:
what's the plan on how to get the chat template to the inference server? I'd think that we 1. read from the model tokenizer directly, 2. if an override is provided on model creation use that, 3. if an override is provided on request use that, but not sure where that ends up being implemented

Reply:
the vllm server does that. if you don't provide a chat template to vllm, it will use the one provided in the hf checkpoint

Reply:
> what's the plan on how to get the chat template to the inference server

For this, i write out the chat template as a CLI arg into vllm server python process by b64 codecs